src/layman/upgrade/upgrade_v2_0.py

```diff
@@ -379,6 +381,25 @@ def migrate_layers():
                 failed_steps.append('normalized_raster_files')
                 logger.error(f'    Fail to move normalized raster files: : \n{traceback.format_exc()}')
 
+        # move layer in DB
+        if geodata_type == settings.GEODATA_TYPE_VECTOR \
```

Comment:
Think about testing new table in name in migration test (either by quering cataloque or layer info key).

Reply:
Implemented by select from table